doc/developers/contributing.rst

```diff
@@ -210,37 +207,25 @@ then submit a "pull request" (PR):
        $ git clone git@github.com:YourLogin/scikit-learn.git
        $ cd scikit-learn
 
-4. Install library in editable mode::
+4. Install the development dependencies::
+
+       $ pip install numpy scipy cython pytest flake8
```

Comment:
`joblib`? Also, I find it suboptimal and hard to maintain that we basically have this section twice in our documentations.

Reply:
I think a little bit of redundancy in our docs is worth it if it avoids users going through the rabbit holes of clicking into our multiple "see *link* for details".